rotkehlchen/chain/ethereum/adex/typing.py

```diff
@@ -0,0 +1,310 @@
+from dataclasses import dataclass
+from enum import Enum
+from typing import Any, Callable, Dict, List, NamedTuple, Optional, Tuple, Union
+
+from eth_typing.evm import ChecksumAddress
+from eth_utils.typing import HexStr
+
+from rotkehlchen.accounting.structures import Balance
+from rotkehlchen.fval import FVal
+from rotkehlchen.typing import Timestamp
+
+AdexEventDBTuple = (
+    Tuple[
+        str,  # tx_hash
+        str,  # address
+        str,  # identity_address
+        int,  # timestamp
+        str,  # type
+        str,  # pool_id
+        str,  # amount
+        str,  # usd_value
+        Optional[str],  # bond_id
+        Optional[int],  # nonce
+        Optional[int],  # slashed_at
+        Optional[int],  # unlock_at
+        Optional[str],  # channel_id
+    ]
+)
+
+
+class EventType(Enum):
+    """Supported events"""
+    BOND = 1
+    UNBOND = 2
+    UNBOND_REQUEST = 3
+    CHANNEL_WITHDRAW = 4
+
+    def __str__(self) -> str:
```

Comment:
Why go into the trouble of having `pretty_name` and `__str__`. The original protocol name is already in the variale name  of the EventType.

I would just keep it simple and keep pretty_name for _str__

Reply:
Fixed it. Please resolve when you read it.

You would probaly need to wipe the DB table with adex events after this since the event type is changed. ....